builds/misc/packages.yaml

```diff
@@ -105,8 +105,6 @@ jobs:
           echo "##vso[task.setvariable variable=PACKAGE_OS;]$(os)"
           echo "##vso[task.setvariable variable=PACKAGE_DOCKER_IMAGE;]$(dockerImage)"
```

Comment:
Do you know what `PACKAGE_DOCKER_IMAGE` is used for?  I didn't find it used anywhere, and I'm not sure `$dockerImage` is defined, either.